package.json

```diff
@@ -1,117 +1,120 @@
-    "prebuild": "npm run clean:build",
-    "test": "npm run lint && npm run test:unit",
-    "test:unit": "cross-env NODE_ENV=test mocha",
-    "test:unit:debug": "cross-env NODE_ENV=test mocha --inspect --debug-brk",
-    "test:coverage": "cross-env NODE_ENV=test BABEL_ENV=coverage nyc mocha && nyc report -r lcovonly",
-    "test:coverage:html": "cross-env NODE_ENV=test BABEL_ENV=coverage nyc mocha && nyc report --reporter=html"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/STORIS/react-scrollbar-size.git"
-  },
-  "keywords": [
-    "react",
-    "scrollbar"
-  ],
-  "author": "STORIS",
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/STORIS/react-scrollbar-size/issues"
-  },
-  "homepage": "https://github.com/STORIS/react-scrollbar-size#readme",
-  "peerDependencies": {
-    "react": "^15.3.0 || ^16.0.0"
-  },
-  "dependencies": {
-    "babel-runtime": "^6.26.0",
-    "prop-types": "^15.6.0",
-    "react-event-listener": "^0.5.1",
-    "stifle": "^1.0.2"
-  },
-  "devDependencies": {
-    "babel-cli": "^6.26.0",
-    "babel-core": "^6.26.0",
-    "babel-eslint": "^8.0.2",
-    "babel-plugin-istanbul": "^4.1.5",
-    "babel-plugin-transform-class-properties": "^6.24.1",
-    "babel-plugin-transform-object-rest-spread": "^6.26.0",
-    "babel-plugin-transform-react-remove-prop-types": "^0.4.12",
-    "babel-plugin-transform-runtime": "^6.23.0",
-    "babel-preset-env": "^1.6.1",
-    "babel-preset-react": "^6.24.1",
-    "babel-register": "^6.26.0",
-    "chai": "^4.0.0",
-    "cross-env": "^5.0.0",
-    "dotenv": "^4.0.0",
-    "enzyme": "^2.8.2",
-    "eslint": "^4.16.0",
-    "eslint-config-airbnb": "^16.1.0",
-    "eslint-config-prettier": "^2.9.0",
-    "eslint-plugin-import": "^2.8.0",
-    "eslint-plugin-jsx-a11y": "^6.0.3",
-    "eslint-plugin-mocha": "^4.11.0",
-    "eslint-plugin-prettier": "^2.5.0",
-    "eslint-plugin-react": "^7.6.1",
-    "fs-extra": "^3.0.1",
-    "jsdom": "^11.0.0",
-    "mocha": "^3.4.2",
-    "nyc": "^10.3.2",
-    "prettier": "^1.10.2",
-    "react": "^15.5.4",
-    "react-dom": "^15.5.4",
-    "react-test-renderer": "^15.5.4",
-    "rimraf": "^2.6.1",
-    "sinon": "^2.3.2"
-  },
-  "browserslist": [
-    "> 1%",
-    "last 2 versions"
-  ],
-  "babel": {
-    "presets": [
-      "env",
-      "react"
-    ],
-    "plugins": [
-      "transform-class-properties",
-      "transform-object-rest-spread",
-      "transform-runtime"
-    ],
-    "env": {
-      "coverage": {
-        "plugins": [
-          "istanbul"
-        ]
-      },
-      "release": {
-        "plugins": [
-          "transform-react-remove-prop-types"
-        ]
-      }
-    }
-  },
-  "nyc": {
-    "include": [
-      "src/**/*.js",
-      "src/**/*.jsx"
-    ],
-    "exclude": [
-      "**/*.spec.js",
-      "**/*.spec.jsx"
-    ],
-    "sourceMap": false,
-    "instrument": false
-  }
+	"name": "react-scrollbar-size",
```

Comment:
Conversion from spaces to tabs on the json file seems unnecessary and its also making it very difficult to detect what was actually changed since everything shows up as a diff.